ENVIRONMENT.md

```diff
@@ -31,6 +31,9 @@ systemctl:
 * `$SYSTEMCTL_FORCE_BUS=1` — if set, do not connect to PID1's private D-Bus
   listener, and instead always connect through the dbus-daemon D-bus broker.
 
+* `$SYSTEMCTL_FORCE_LOCAL=1` — if set, do not connect through the dbus-daemon
```

Comment:
hmm, I think this should be called SYSTEMCTL_FORCE_PRIVATE really. In all other contexts we refer to the non-dbus-daemon connection as the "private systemd socket", and we should reuse the same naming scheme here, too.